flash_attn/cute/mask.py

```diff
@@ -135,17 +135,23 @@ def apply_mask(
                     # Convert to absolute column index
                     global_col_idx = thr_col_offset + col_idx_local + n_block * self.tile_n
 
-                    cond = cutlass.Boolean(
-                        mask_mod(
-                            batch_idx,
-                            head_idx,
-                            tScS_mn[r, 0][0] + m_block * self.tile_m,
-                            thr_col_offset + t0ScS_mn[0, col][1] + n_block * self.tile_n,
-                            self.seqlen_q,
```

Comment:
Semantic change, I remove seqlen_q and seqlen_kv and instead switch to the flex mask_mod api